src/languages/es.ts

```diff
@@ -2354,7 +2354,7 @@ export default {
             importedFromAccountingSoftware: 'Impuestos importadas desde',
         },
         emptyWorkspace: {
-            title: 'Crea un espacio de trabajo',
+            title: 'Crea un nuevo espacio de trabajo',
```

Comment:
is this translation verified?

Reply:
Same comment - I think we can just keep the existing copy that we had before, sorry for the misleading mockup! Let's undo this change.